.github/workflows/package-debian.yml

```diff
@@ -41,26 +36,49 @@ jobs:
     container:
       image: ghcr.io/mariusbgm/sil-kit-ci-ubuntu-20.04:main
     steps:
-      - name: Checkout
+
+      - name: Checkout (sil-kit-pkg)
+        uses: actions/checkout@v4
+        with:
+          path: sil-kit-pkg
+
+      - name: Checkout (sil-kit)
         uses: actions/checkout@v4
         with:
-          submodules: true
+          repository: ${{ inputs.silkit_source_repo }}
+          submodules: recursive
+          path: sil-kit
+          ref: ${{ inputs.silkit_source_ref }}
+          github-server-url: https://github.com
+
       - name: Install build dependencies
         run: sudo apt install -y debhelper dpkg-dev devscripts dh-make zip
+
       - name: Install cmake
         run: |
-          sh ./.github/actions/fetch_cmake.sh
+          sh sil-kit-pkg/.github/actions/fetch_cmake.sh
+
       - name: Build
         id: build
         env:
-          SILKIT_SOURCE_URL: ${{ inputs.silkit_source_url }}
-          SILKIT_REVISION: ${{ inputs.silkit_source_ref }}
-          SILKIT_PKG_URL: ${{ inputs.silkit_pkg_url }}
+          SILKIT_SOURCE_URL: ${{ github.workspace }}/sil-kit
+          SILKIT_PKG_URL: ${{ github.workspace }}/sil-kit-pkg
           DEBFULLNAME: ${{ inputs.debian_fullname }}
           DEBEMAIL: ${{ inputs.debian_fullname }}
           CI_RUN: "1"
         run: |
-          sh ./.github/actions/build_deb.sh 20.04
+          sh sil-kit-pkg/.github/actions/build_deb.sh 20.04
+
+      - name: Test Package
+        id: test
+        env:
+          DEB_DIR: ${{ github.workspace }}
+          PKG_DIR: ${{ github.workspace }}/sil-kit-pkg/
+        run: |
+          python3 sil-kit-pkg/tests/test_deb.py \
+          --deb-directory "$DEB_DIR" \
+          --package-directory "$PKG_DIR"
+
       - name: Artifact
         env:
           ARTIFACT_NAME: libsilkit4_${{ steps.build.outputs.silkit_version }}-${{ steps.build.outputs.silkit_debian_revision }}_${{ inputs.debian_arch }}
```

Comment:
Shouldn't this more like `ubuntu-20.04-deb.zip` ? the debian artifacts inside of the zip are already named `libsilkit4*`

Reply:
New format is `SilKit_${silkit_source_ref}-ubuntu-20.04.zip`

Reply:
The name was too verbose. changed to silkit-ubuntu-20.04-deb.zip 👍